arviz/tests/base_tests/test_plots_matplotlib.py

```diff
@@ -370,7 +374,17 @@ def test_plot_kde_cumulative(continuous_model, kwargs):
     assert axes
 
 
-@pytest.mark.parametrize("kwargs", [{"kind": "hist"}, {"kind": "kde"}])
+@pytest.mark.parametrize(
+    "kwargs",
+    [
+        {"kind": "hist"},
+        {"kind": "kde"},
+        {"is_circular": False},
+        {"is_circular": True},
+        {"is_circular": "radians"},
+        {"is_circular": "degrees"},
+    ],
+)
 def test_plot_dist(continuous_model, kwargs):
     axes = plot_dist(continuous_model["x"], **kwargs)
```

Comment:
I was quite shocked by the low coverage given that there are tests for everything, I was expecting something close to 100% diff coverage. I think the issue (and same happens with bokeh) is that the variables in the model are continuous, so the kde is plotted. I am not sure if it would be better to add a new test of divide the fixture into two so all combinations are tested